src/components/tasks/card/index.tsx

```diff
@@ -53,6 +52,11 @@ const Card: FC<Props> = ({
     if (state?.isLoggedIn) {
       getTaskTags();
     }
+    if (cardDetails.assignee) {
+      getAssigneeProfilePic(cardDetails.assignee).then((res) => {
```

Comment:
[nitpick] Wondering if there is a specific reason for NOT using `await` here?

Reply:
We can't wait for the image to load, then load the task. First the task gets loaded then the images keep loading in the background

Reply:
Your image will load async and your code will be loaded by the time the image loads which means your component will be visible to the user. images won't load in that but the `res` of the component will load. 

also, the video is recorded in fast 3g throttled mode.

I will request you to make changes and test once and let me know if it creates any issue


https://user-images.githubusercontent.com/64726586/230130458-1530c793-3e33-4134-a845-62c2adbf1e06.mp4

Reply:
@vvaibhavdesai I guess this is what he want to do load images async, and a show placeholder meanwhile.

Correct me If a wrong @sahsisunny

Reply:
> @vvaibhavdesai I guess this is what he want to do load images async, and a show placeholder meanwhile.
> 
> Correct me If a wrong @sahsisunny

Yes

Reply:
@shmbajaj that can be done irrespective. Nothing to do with awaiting here.
Also, this is what he said 
![image](https://user-images.githubusercontent.com/64726586/230268256-eac45e2e-0630-4815-bcea-de794e62b2f1.png)
and the video I shared is the code written with `async` and `await`